narwhals/series.py

```diff
@@ -2148,6 +2148,50 @@ class SeriesStringNamespace:
     def __init__(self, series: Series) -> None:
         self._narwhals_series = series
 
+    def strip_chars(self, characters: str | None = None) -> Series:
+        r"""
+        Remove leading and trailing characters.
+
+        Arguments:
+            characters: The set of characters to be removed. All combinations of this set of characters will be stripped from the start and end of the string. If set to None (default), all leading and trailing whitespace is removed instead.
+
+        Examples:
+            >>> import pandas as pd
+            >>> import polars as pl
+            >>> import narwhals as nw
+            >>> data = {"fruits": ["apple", "\nmango"]}
+            >>> df_pd = pd.DataFrame(data)
+            >>> df_pl = pl.DataFrame(data)
+
+            We define a dataframe-agnostic function:
+
+            >>> @nw.narwhalify
+            ... def func(df):
+            ...     return df.with_columns(stripped=nw.col("fruits").str.strip_chars())
+
+            We can then pass either pandas or Polars to `func`:
+
+            >>> func(df_pd)
+                fruits stripped
+            0    apple    apple
+            1  \nmango    mango
+
+            >>> func(df_pl)  # doctest: +NORMALIZE_WHITESPACE
+            shape: (2, 2)
+            ┌────────┬──────────┐
+            │ fruits ┆ stripped │
+            │ ---    ┆ ---      │
+            │ str    ┆ str      │
+            ╞════════╪══════════╡
+            │ apple  ┆ apple    │
+            │        ┆ mango    │
+            │ mango  ┆          │
+            └────────┴──────────┘
+        """
```

Comment:
Here we should exemplify the behaviour on a series, not on an Expr 🙃

Reply:
Oops ! 😁